README.md

```diff
@@ -207,6 +207,12 @@ ROM files contain the game itself. Each system has a unique file extension to de
 
 Sometime ROMs from these systems use different extensions, e.g. `.gen` for Genesis, `.bin` for Atari, etc. Please rename the ROMs to use the aforementioned extensions in these cases.
 
+You can import your ROMs by [`retro.import`](https://github.com/openai/retro/blob/master/scripts/import.py). 
```

Comment:
might want to change the phrasing a bit, e.g. "You can import your ROMs using..." or "You can import your ROMs with...". It seems a little off as is.